chemprop/cli/train.py

```diff
@@ -476,6 +480,14 @@ def validate_train_args(args):
 
 
 def main(args):
+    command_config_path = args.output_dir / "config.json"
+    with open(command_config_path, "w") as f:
+        config = deepcopy(vars(args))
```

Comment:
Why do you use deepcopy?

Reply:
Thanks for the question! I converted the `Path` to `str` before saving it as JSON as `Path` cannot be serialized. However, `vars(args))` does not make a copy and will cause the `Path` stored in args also turn into `str` and causes problem downstream as later codes assume it's `Path`.